clio-server/src/main/scala/org/broadinstitute/clio/server/service/ServerService.scala

```diff
@@ -155,13 +155,16 @@ object ServerService {
   val RecoveryMaxBulkSize: Long = 1000L
 
   def apply(
```

Comment:
Can scrap this one too; `ClioServer` can just pass `ClioServerConfig.Version.value` to the constructor when it builds the service